library/src/morpheus-master/sass/modules/tool/syllabus/_syllabus.scss

```diff
@@ -51,11 +51,11 @@
 
   .ui-accordion .ui-accordion-header {
     overflow: auto !important;
-    background-color: #D9EEF7;
+    background-color: var(--infoBanner-bgcolor);
```

Comment:
I'd like to see these a more generic colour, like grey. But maybe this could be a different PR.

Reply:
this is overridden later in the file anyway so i'll delete this rule.